### DISCUSSION TITLE
How Can I fix this SEB error?

### DISCUSSION THREAD

**Discussion by Krangl** [SafeExamBrowser/seb-win-refactoring#600] [Q&A]
Seb failed to start a New session.Hello,
I've installed SEB 3.5.0
![16817203675274543137926973661983](https://user-images.githubusercontent.com/130971809/232430131-783d1aee-6174-4d80-a5b3-d86389e1911e.jpg)
 on several time and can't start the application. #456 #94 #566 #140

**Comment by dbuechel** [comment]
Posting the same issue various times generates a lot of overhead for us and pollutes other, potentially completely unrelated discussions and issues. It also doesn't make us pay more attention to a particular support request, hence please abstain from doing so in the future.

We always try to give our best to handle every support request or question we receive as soon as possible, but it may take some time every once in a while. We're publicly funded and thus have very limited financial / personal resources and can't always take care of everything at once.

**Comment by dbuechel** [comment]
Now in order to investigate what your particular issue is here, it would be great if you could upload the runtime and client log files which are mentioned in the error message above.
